frame/system/src/lib.rs

```diff
@@ -1123,7 +1104,7 @@ impl<T: Trait> Module<T> {
 	#[cfg(any(feature = "std", test))]
 	pub fn set_block_limits(weight: Weight, len: usize) {
```

Comment:
Not related to your PR but I think you can agree that this function's api is a bit screwed now.

Reply:
>  Do you mean that the name clashes a bit with the config parameters


Yeah and I see the name now reflects that. Indeed not important as it is only test.